mailpoet/assets/js/src/automation/integrations/woocommerce/steps/order-cancelled/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { __, _x } from '@wordpress/i18n';
+import { StepType } from '../../../../editor/store';
+import { Icon } from './icon';
+
+const keywords = [__('woocommerce', 'mailpoet'), __('order', 'mailpoet')];
```

Comment:
@websupporter While testing I noticed I couldn't find the triggers by searching for `cancelled`, `completed` or `created` which seems like something we should fix. Do you know if there's a reason why the search doesn't also search the titles of triggers?

Reply:
Ah! Thanks. Fixed it in https://github.com/mailpoet/mailpoet/pull/5247/commits/3135b8c6836f86c330164e746875d9a931733870

> Do you know if there's a reason why the search doesn't also search the titles of triggers

We first used only the title but then at some point the title needed to be able to return `JSX.Element` and we introduced keywords.